src/CMakeLists.txt

```diff
@@ -1,8 +1,19 @@
 find_package(PkgConfig REQUIRED)
-pkg_check_modules(EventdEvent REQUIRED libeventd-event>=0.3)
+pkg_check_modules(EventdEvent REQUIRED libeventd>=0.3)
```

Comment:
Oh, yeah, sure. I still consider eventd unstable enough to just be version-less. :-D